src/practise/SlidingWindow.java

```diff
@@ -20,4 +21,23 @@
     return results;
   }
 
+  public int[] sumSlidingWindow(int[] arr, int groupSize) {
+    int[] results = new int[(arr.length - groupSize) + 1];
+    int sum = 0;
+    int index = 0;
+    for (int i = 0; i < arr.length; i++) {
+      if (i < groupSize) {
+        sum = sum + arr[i];
+        if (i == groupSize - 1) {
+          results[index++] = sum;
+        }
+        continue;
+      }
+      sum = sum - arr[i - groupSize];
+      sum = sum + arr[i];
+      results[index++] = sum;
```

Comment:
Sum is being pushed to results in first condition as well. Can that be more concise?